### PR TITLE
Install Rust into our docker images

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

File: cryptography-manylinux/Dockerfile-manylinux1 [hunk 1]
```diff
@@ -14,3 +14,6 @@ ADD openssl-version.sh /root/openssl-version.sh
 RUN sh install_openssl.sh manylinux1
 ADD install_virtualenv.sh /root/install_virtualenv.sh
 RUN sh install_virtualenv.sh manylinux1
+
+RUN curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable
+RUN source $HOME/.cargo/env
```
Comment (reaperhulk) [inline cryptography-manylinux/Dockerfile-manylinux1:19]: What are you expecting this to do?

Comment (alex) [reply cryptography-manylinux/Dockerfile-manylinux1:19]: Put something on the path, are env vars not persisted across RUNs?

Comment (reaperhulk) [reply cryptography-manylinux/Dockerfile-manylinux1:19]: They are not. If you want this in PATH you need to add it to a file sourced when the shell starts

**Comment by alex** [comment]
FWIW, my plan is once we know what version we want to target as our MSRV, and that version is in a distro (e.g. debian) I'll switch that builder from rustup.rs to installing with apt.

**Pull request merged**